LOG.tbasicu

```diff
@@ -216,124 +220,123 @@ Type tLogThis
     
       end if
       
-      if Bit_Get(logWhere,5) then ' %LT_ToLogFile is SET
-        if me.RotateLog = %FALSE then allOK = allOK and %FALSE
+      if me.private_isFlagSet(logWhere, %LT_ToLogFile) then
+        if me.RotateLog = %FALSE then allOK = %FALSE
         local sb as string
         sb = sDTF & sTagF & logWhatTitleFile & me.titleSeparatorFile & logWhat & me.endLineFile
-        successFile = FILE_Append(me.logFile,sb)
-        if successFile <> 0 Then allOK = allOK and %FALSE
-        Reset successFile
+        
+        allOK = (FILE_Append(me.logFile,sb) = 0)
       End If
-      
-      if Bit_Get(logWhere,6) then ' %LT_ToStatusBar is SET
+            
+      if me.private_isFlagSet(logWhere, %LT_ToStatusBar) then
         if me.statusbarhWnd <> 0 and me.statusbarctrlID <> 0 and me.statusbarlPart <> 0 Then ' Function SetStatusbar was previously called
           StatusBar_SetText(me.statusbarhWnd, me.statusbarctrlID, logWhatTitle & me.titleSeparator & logWhat, me.statusbarlPart)
         Else
-          allOK = allOK and %FALSE
+          allOK = %FALSE
         end if
       End If
     
     end If
     
-    if allOK then
-      Return %TRUE
-    Else
+    if not allOK then
       if isDebug or (not isDebug and verbose) then MsgBoxw(Null,"Error in function :" & Function_Name)
-      Return %FALSE
     end if
+    
+    Return allOK
```

Comment:
No need for Return %TRUE and Return %FALSE, we have all we need in allOK

Reply:
yes !